code/game/gamemodes/clock_cult/clock_objects.dm

```diff
@@ -194,7 +194,7 @@
 	user << "<i>Inath-Neq: </i>[!clockwork_generals_invoked["inath-neq"] ? "<font color='green'><b>Ready</b></font>" : "<span class='boldannounce'>Recovering</span>"]"
 
 /obj/item/clockwork/slab/proc/show_guide(mob/living/user)
-	var/text = "<font color=#BE8700 size=3><b><center>Chetr nyy hageh�guf naq ubabe Ratvar.</center></b></font><br><br>\
+	var/text = "<font color=#BE8700 size=3><b><center>Chetr nyy hagehguf naq ubabe Ratvar.</center></b></font><br><br>\
```

Comment:
Who knows, the exact same line shows as changed for no apparent reason in the other PR